include/wx/graphics.h

```diff
@@ -373,82 +551,6 @@ class WXDLLIMPEXP_CORE wxGraphicsPath : public wxGraphicsObject
 extern WXDLLIMPEXP_DATA_CORE(wxGraphicsPath) wxNullGraphicsPath;
 
 
-// Describes a single gradient stop.
```

Comment:
It's still a bit annoying that this has moved around. We can use `git diff --color-moved`, of course, but e.g. GitHub doesn't detect moves, so if it were possible to move it back, it would result in a better diff here.